runners/jet-experimental/build.gradle

```diff
@@ -0,0 +1,85 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
```

Comment:
@adude3141 should there be some publishing config here to get the artifactId set up?

Reply:
Not quite sure, if I understand you correctly, @kennknowles 

I think, you are referring to archiveBaseName here. It is usually not necessary to setup on project level. Only if the default seems improper, we are ab le to overwrite.

E.g. in this case, the folder path is `runners:jet-experimental` which will result in artefact named something about `beam-runners-jet-experimemtal`. Now, we could decide that this is not wanted as after switching from experimental mode we would like to keep artifact named consistent. So we could add an
```
archivesBaseName = 'beam-runners-jet'
```

More likely in this though, would be thet we would like to drop that `experimental`on the folder name, i.e. move `runners/jet-experimental` to `runners/jet` and set

```
archivesBaseName = 'beam-runners-jet-experimental'
```
for the time being.

(Hope I got it right)

Reply:
OK, I see. I did not quite understand that the archive base name was so well automated. That is great.